### PR TITLE
Splitup element

### PR DESCRIPTION
**Pull request by svenpopping**

### CODE REVIEW

File: lancie-activities.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+    <div id="container" class="layout horizontal wrap start-justified">
+      <template is="dom-repeat" items="[[activities]]" as="activity">
+        <lancie-activity class="lancie-block" data="{{activity}}" style$="{{_calcWidth()}}"></lancie-activity>
+      </template>
+    </div>
+
+  </template>
+</dom-module>
+<script>
```
Comment (JeremybellEU) [inline lancie-activities.html]: The `script` tag should be in the `dom-module`

File: lancie-activities.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  Polymer({
+    is: 'lancie-activities',
+    properties: {
+      data: {
+        type: Array,
+        value: []
+      },
+      json: {
+        notify: true
```
Comment (JeremybellEU) [inline lancie-activities.html]: Does it need to notify? And I think it should be a String, so:
```javascript
json: {
  type: String
}
```

File: lancie-activities.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        padding: 20px 10px;
+        box-sizing: border-box;
+      }
+    </style>
+
+    <iron-ajax auto url="{{json}}" handle-as="json" last-response="{{activities}}" debounce-duration="300"></iron-ajax>
+
+    <div id="container" class="layout horizontal wrap start-justified">
+      <template is="dom-repeat" items="[[activities]]" as="activity">
+        <lancie-activity class="lancie-block" data="{{activity}}" style$="{{_calcWidth()}}"></lancie-activity>
```
Comment (JeremybellEU) [inline lancie-activities.html]: The function `_calcWidth()` will only fire once, but is dependant on the width of the client. Can we have some sort of observer to re-trigger the calculation? Maybe with `iron-media-query`

File: lancie-activity-button.html [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+  </template>
+</dom-module>
+<script>
+Polymer({
+  is: 'lancie-activity-button',
+  properties: {
+    data: {
+      type: Object,
+      value: {},
+      notify:true
```
Comment (JeremybellEU) [inline lancie-activity-button.html]: Notify is probably also not needed here. And the `script` tag.

File: lancie-activity-dialog.html [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+<link rel="import" href="/bower_components/paper-material/paper-material.html">
+<link rel="import" href="/bower_components/paper-icon-button/paper-icon-button.html">
+
+<link rel="import" href="/bower_components/neon-animation/neon-animations.html">
+
+<link rel="import" href="lancie-activity-content.html">
+<link rel="import" href="lancie-activity-sponsor.html">
+
+<dom-module id="lancie-activity-dialog">
+  <!-- <link rel="import" type="css" href="dist/css/lancie-dialog-element.css"/> -->
```
Comment (JeremybellEU) [inline lancie-activity-dialog.html]: Left-over import

File: lancie-activity-footer.html [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+          <span class="label">Sponsor</span>
+          <span class="info">{{data.sponsor.name}}</span>
+        </li>
+      </ul>
+    </div>
+  </template>
+</dom-module>
+<script type="text/javascript" src="/bower_components/moment/min/moment.min.js"></script>
+<script>
+  Polymer({
```
Comment (JeremybellEU) [inline lancie-activity-footer.html]: Missing `'use strict';` and must be within `dom-module`

**Comment by JeremybellEU** [comment]
I will do a visual inspection later.

Also the JSON flow can be optimized. We are now passing all `data`, whereas we might just want to pass a sub-property like `sponsor` or `startingTime`. cc @skamoen

File: lancie-activities.html (hunk 3 above)
Comment (svenpopping) [inline lancie-activities.html]: Yeah it should be re-triggered when resizing the width of the client. I created a issue for that #6.

**Comment by skamoen** [comment]
You can create/update the issue of filtering data in the api repo. @martijn9612 is currently working on the web systems.

**Comment by JeremybellEU** [comment]
LGTM

**Pull request merged**